Gems/Atom/Asset/Shader/Code/Source/Platform/Common/Clang/atom_asset_shader_static_clang.cmake

```diff
@@ -6,7 +6,11 @@
 #
 #
 
-set(LY_COMPILE_OPTIONS
+if(MSVC)
+    set(LY_COMPILE_OPTIONS PRIVATE /EHsc)
+else()
+    set(LY_COMPILE_OPTIONS
     PRIVATE
         -fexceptions # The macro PYBIND11_EMBEDDED_MODULE uses a try catch block
-)
+    )
+endif()
```

Comment:
This comment also wants either copying to both sides or above the if/else

Reply:
done